build.sbt

```diff
@@ -64,15 +64,18 @@ lazy val membership = project
   .in(file("membership"))
   .settings(stdSettings("zio-membership"))
   .settings(
+    mainClass in (Compile, run) := Some("zio.membership.Main"),
     libraryDependencies ++= Seq(
-      "dev.zio"                %% "zio"                     % "1.0.0-RC17",
-      "dev.zio"                %% "zio-streams"             % "1.0.0-RC17",
-      "dev.zio"                %% "zio-nio"                 % "0.4.0",
-      "dev.zio"                %% "zio-macros-core"         % "0.6.0",
-      "com.lihaoyi"            %% "upickle"                 % "0.8.0",
-      "org.scala-lang.modules" %% "scala-collection-compat" % "2.1.2",
-      "dev.zio"                %% "zio-test"                % "1.0.0-RC17" % Test,
-      "dev.zio"                %% "zio-test-sbt"            % "1.0.0-RC17" % Test
+      "dev.zio"     %% "zio"             % "1.0.0-RC17",
+      "dev.zio"     %% "zio-streams"     % "1.0.0-RC17",
+      "dev.zio"     %% "zio-nio"         % "0.4.0",
+      "dev.zio"     %% "zio-macros-core" % "0.6.0",
```

Comment:
I believe we're good at these for now. However, they might be tricky in the future, and I'm not sure whether the benefit accommodates the cost.

Reply:
What do you mean?